aptoide-database/src/main/java/cm/aptoide/pt/database/room/NotificationDao.java

```diff
@@ -0,0 +1,36 @@
+package cm.aptoide.pt.database.room;
+
+import androidx.room.Dao;
+import androidx.room.Insert;
+import androidx.room.Query;
+import io.reactivex.Observable;
+import io.reactivex.Single;
+import java.util.List;
+
+import static androidx.room.OnConflictStrategy.REPLACE;
+
+@Dao public interface NotificationDao {
+
+  @Query("SELECT * from NOTIFICATION where type IN (:notificationType) AND dismissed BETWEEN :startingTime AND :endTime ")
+  Single<List<RoomNotification>> getDismissed(Integer[] notificationType, long startingTime,
```

Comment:
Why Integer[], seems like we always pass a single int.

Reply:
For the social policies we are passing several like (like, comment, share, etc). I know that this is related with old code, but for some reason it was left in the app (future suport ? dont know).

Reply:
Social policies are not used anymore, but lets not make the changes now to reduce the tests scope. Please create a ticket to investigate and remove code related to SocialPolicies (also code related to AptoideNotifications types that are not used anymore) and I'll consider this resolved.